frontend/src/components/explorer/mapViewer/SidebarDataPanels.vue

```diff
@@ -7,28 +7,45 @@
           <i>{{ currentMap.name }}</i>
         </p>
       </header>
-      <div v-if="currentMap.reactionList && missingReactionList.length > 0"
+      <div v-if="dim==='2d' && currentMap.reactionList && missingReactionList.length > 0"
            class="card-content p-4">
-        <div class="content mb-0">
+        <div v-if="currentMap.mapReactionIdSet.length == 1" class="content mb-0">
           Please note that {{ missingReactionList.length }}
           of the reactions in the {{ currentMap.type }} are not shown on the map.
           <a @click="showModal = true"> See comparison </a>
         </div>
+        <div v-else class="content mb-0">
+          Please note that {{ missingReactionList.length }} of the reactions in the
+          {{ currentMap.type }} are not shown on any of the {{ currentMap.name }} maps.
+          <a @click="showModal = true"> See comparison </a>
+        </div>
         <div v-if="showModal" class="modal is-active">
           <div class="modal-background" @click="closeModal"></div>
           <div class="modal-content p-5 column is-6-fullhd is-8-desktop is-10-tablet is-full-mobile
           has-background-white">
             <h4 class="title is-size-4 m-0 mb-2"> List of missing and total reactions on the map </h4>
-            <p class="pb-4">
+            <p v-if="currentMap.mapReactionIdSet.length == 1" class="pb-4">
               There are {{ missingReactionList.length }} reactions not shown on the map. Some reactions
               are missing as the {{ currentMap.type }} is being updated much more often than the maps.
               Also, as the maps are manually curated, occasionally some reactions cannot be added.
               The number of reactions shown are {{ mapReactionList.length }}.
             </p>
+            <p v-else class="pb-4">
+              There are {{ missingReactionList.length }} reactions not shown on any of the {{ currentMap.name }} maps.
+              {{ currentMap.name }} is split into multiple maps due to its size and the number of missing
```

Comment:
Maybe better to change to `The 2D map of {{ currentMap.name }} is split into multiple maps`. It is Cytosol map that is split, not Cytosol itself.

Reply:
What about end the sentence at `Cytosol is split into multiple maps due to its size.`, since the following explanation is a separate thing. Then followed by `The number of missing reaction displayed is the missing reactions when the reactions in all Cytosol maps are combined`

Reply:
Good suggestions! Incorporated them with the latest commit :+1: